promql/engine.go

```diff
@@ -1090,6 +1090,10 @@ type EvalNodeHelper struct {
 	rightSigs    map[string]Sample
 	matchedSigs  map[string]map[uint64]struct{}
 	resultMetric map[string]labels.Labels
+
+	// MatrixSelector shall ignore sample(s) on the left bound, i.e., (leftTimestamp, rightTimestamp].
+	isLeftBoundValid bool
+	leftBound        int64
```

Comment:
You don't really need these two, do you? (In this PR, you only set them and never read them.)

Reply:
Thanks, I shall remove them.